enterprise/coderd/coderd.go

```diff
@@ -1128,3 +1155,19 @@ func (api *API) runEntitlementsLoop(ctx context.Context) {
 func (api *API) Authorize(r *http.Request, action policy.Action, object rbac.Objecter) bool {
 	return api.AGPL.HTTPAuth.Authorize(r, action, object)
 }
+
+func (api *API) setupPrebuilds(entitled bool) (agplprebuilds.ReconciliationOrchestrator, agplprebuilds.Claimer) {
+	enabled := api.AGPL.Experiments.Enabled(codersdk.ExperimentWorkspacePrebuilds)
```

Comment:
maybe call it `prebuildsExperimentEnabled` instead of `enabled` to avoid confusion?
Because in upstream code `enabled` means `feature is enabled`, but here it means `experiment is enabled`.

Also maybe rename `entitled` to `featureEnabled`?

Reply:
No need to ask permission for changes things like this

Whatever is both a) clear and b) consistent will usually be accepted in code-review.

Reply:
not relevant in this PR anymore